exchange_calendars/exchange_calendar_aixk.py

```diff
@@ -111,10 +127,7 @@
 )
 
 IndependenceDayHoliday = Holiday(
-    "Independence Day",
-    month=12,
-    day=17,
-    observance=next_monday,
+    "Independence Day", month=12, day=17, observance=next_monday, end_date="2022"
```

Comment:
A nitpick , but could you stick this comma in and rerun black, then I think we're all good.
```suggestion
    "Independence Day", month=12, day=17, observance=next_monday, end_date="2022",
```